### PR TITLE
pycloudstack: support to add driver in xml template

### PR DESCRIPTION
**Pull request by ruomengh**
1. Support to add driver in xml template for network perf test
2. Update vtpmtd binary name in vmparam.py

### CODE REVIEW

File: utils/pycloudstack/pycloudstack/virtxml.py [hunk 1]
```diff
@@ -626,6 +668,16 @@ def set_hugepage_params(self, hugepage_size):
             {"unit": f"{unit}", "size": f"{size}"})
         self.save()
 
+    def set_driver(self, driver):
```
Comment (kenplusplus) [inline utils/pycloudstack/pycloudstack/virtxml.py:671]: set_driver sound a little confusing, what driver?

File: utils/pycloudstack/pycloudstack/vmparam.py [hunk 2]
```diff
@@ -35,7 +35,7 @@
 BIOS_OVMF_VTPM = "/usr/share/tdx-vtpm/OVMF.fd"
 
 # vTPM TD binary file
-VTPM_PATH = "/usr/share/tdx-vtpm/final.bin"
+VTPM_PATH = "/usr/share/tdx-vtpm/vtpmtd.bin"
```
Comment (kenplusplus) [inline utils/pycloudstack/pycloudstack/vmparam.py:38]: could we split this into a new patch, vtpmtd.bin is belong to driver?

**Review by kenplusplus — CHANGES_REQUESTED** [review]

File: utils/pycloudstack/pycloudstack/vmparam.py (hunk 2 above)
Comment (ruomengh) [reply utils/pycloudstack/pycloudstack/vmparam.py:38]: OK, I'll close this PR and open a new one to separate the changes.

**Comment by ruomengh** [comment]
Will raise new PRs to separate the changes according to review comment.

**Pull request closed without merging**